titus-server-master/src/main/java/com/netflix/titus/master/scheduler/TaskPlacementFailure.java

```diff
@@ -54,6 +54,11 @@
          */
         LaunchGuard,
 
+        /**
+         * Task not launched on an agent, as it runs the maximum allowed number of containers.
+         */
+        AgentContainerLimit,
```

Comment:
add this new kind to `FailureKind.IGNORED_FOR_OPPORTUNISTIC_SCHEDULING` since it should not cause opportunistic behavior to change